### PR TITLE
Replace Wercker with Travis-CI

### PR DESCRIPTION
**Pull request by taion809**
Hello,
This PR introduces two things, the first is that phpcs is added to require-dev.

The other is that this includes travis-ci in place of wercker. The reasoning behind this change is that php is not a first class citizen on wercker and relies upon userland "boxes" to build environments (which isn't the worst thing in the world). The other, and more important, reason is that because this library is supporting php 5.3+ it should have a better build matrix to report failure across the various versions of php in production today.  For example tests fail on php7 at the moment but wercker will report success due to the fact it's only testing 5.6

### CODE REVIEW

**Comment by taion809** [comment]
link to build on travis https://travis-ci.org/taion809/php-rql

**Comment by mbrevda** [comment]
"Boxes" on wercker are simply Docker containers, making dev/build/production parity much simpler. That being said, the lack of build metrix is unfortunate and is a great reason to move to Travis.

**Comment by danielmewes** [comment]
Hi @taion809, thanks for the PR first of all.
Sorry I haven't gotten around to look at it yet. Just wanted to let you know that I haven't forgotten. I'll hopefully get a chance to review it soon.

File: tests/Functional/JoinTest.php [hunk 1]
```diff
@@ -152,19 +152,19 @@ public function testEqJoinId()
...
+                'right' => array('id' => 'a', 'other' => 1 )
+            ),
         );
 
         $res = $this->db()->table('t1')
             ->eqJoin('id', $this->db()->table('t2'), array('index' => 'other'))
             ->run($this->conn);
 
-        $this->assertEquals($excpected, $this->orderArrayByLeftId($res));
+        $this->assertEquals($excpected, $this->toArray($res->toArray()));
```
Comment (danielmewes) [inline tests/Functional/JoinTest.php]: The ordering of eqJoin is actually undefined. I think both the old and the new code are wrong, and only pass if you're lucky with how the RethinkDB server happened to execute the query. I'll fix this later.

**Comment by danielmewes** [comment]
Merged into `dev`. Thanks again @taion809 !

**Comment by danielmewes** [comment]
Works great.

The tests with newer PHP versions (5.6 and up)) revealed that we were not converting microseconds in ReQL time objects correctly. It took me a few iterations to get the tests passing reliably on all versions, but finally things are looking good: https://travis-ci.org/danielmewes/php-rql/builds/127996608

**Pull request closed without merging**